transform/app/api/routes/solr_utils/solr.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import APIRouter, Query
```

Comment:
We can name this file not solr but e.g. "create_collections". That way all endpoints will have a separate file for themselves

Reply:
I did not know the convention. Corrected.